store/tikv/snapshot.go

```diff
@@ -93,7 +93,6 @@ type KVSnapshot struct {
 		matchStoreLabels []*metapb.StoreLabel
 	}
 	sampleStep uint32
-	txnScope   string
```

Comment:
Is this unused variable redundant or will it be used in the future? cc @Yisaer

Reply:
I think this is introduced by https://github.com/pingcap/tidb/pull/22789, is this ok to be removed? @sticnarf